.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+## Description
+
+Please include a summary of the change, motivation and context
```

Comment:
This is an excessive list to read by a contributor but also to always skip by the maintainer. As a maintainer, I want to see just comments from the contributor. if we keed the text in the body of the section contributors will either delete it (more work for them) or comment inline which is hard to recognize by us.

I would rather put the section description as a hidden comment (not sure if it can be done in .md) or inline with the title e.g.
`## Description: <Describe changes: for bugfix what issue it fixes.  For feature what use-case it solves.>`

Reply:
While I like a lot the short version I miss examples as people might understand different things here.

Reply:
same question!

Reply:
> ## Description: <Describe changes: for bugfix what issue it fixes. For feature what use-case it solves.>

looks enough to me, the current description in the PR seems like too babysitting. 

E.g. the `on a bugfix` contains ` link relevant information about the bug (github issue or slack thread)`. This should be in a separate section for adding links to tracking issues.

Reply:
To give more context, the reason why we started this template is because relevant information (description, motivation and context) isn't being added on PRs. I repeated this list (accordingly) many times across different PRs and repos and hence I think it would be cool to automate that speech over this template.

Reply:
I agree! @pavolloffay let's keep it for now. eventually people will become habitual and then we can edit the template!